<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParseMind - Basic UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblBackendHeader">
      <property name="text">
       <string>Backend Status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblBackendStatus">
      <property name="text">
       <string>Starting...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblOllamaHeader">
      <property name="text">
       <string>LLM / Ollama Status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblOllamaInstalled">
      <property name="text">
       <string>Installed: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblOllamaRunning">
      <property name="text">
       <string>Running: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listModels">
      <property name="toolTip">
       <string>Detected LLM models</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hButtons">
      <item>
       <widget class="QPushButton" name="btnRetryHealth">
        <property name="text">
         <string>Retry Health</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRestartBackend">
        <property name="text">
         <string>Restart Backend</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
